lib/dry/validation/matchers/version.rb

```diff
@@ -1,7 +1,7 @@
 module Dry
   module Validation
     module Matchers
-      VERSION = "0.4.0"
+      VERSION = "0.4.1"
```

Comment:
No need to bump this, we can decide what version the gem should be when it's time to release. After that, I'm good merging this!